medusa/subtitles.py

```diff
@@ -371,9 +410,15 @@ def download_subtitles(tv_episode, video_path=None, subtitles=True, embedded_sub
             logger.info(u'No subtitles found for %s', os.path.basename(video_path))
             return []
 
+        warn_user = False
+        needed_guess = {'format', 'series', 'year', 'episode', 'season', 'video_codec', 'release_group'}
         min_score = get_min_score()
         scored_subtitles = score_subtitles(subtitles_list, video)
         for subtitle, score in scored_subtitles:
+            missing_guess = list(needed_guess - subtitle.get_matches(video))
```

Comment:
This operation is cpu intensive and we already called it when we scored the subtitle.